osg_configure/configure_modules/gratia.py

```diff
@@ -614,95 +419,6 @@ def _configure_htcondor_ce_probe(self):
-                # os.path.samefile will die if the paths don't exist so check that explicitly (SOFTWARE-1735)
-                if not os.path.exists(data_folder):
-                    self.log("DataFolder setting in %s (%s) points to a nonexistant location" % (
-                    config_location, data_folder),
-                             level=logging.ERROR)
-                    valid = False
-                elif not os.path.exists(history_dir):
-                    self.log("Condor PER_JOB_HISTORY_DIR %s points to a nonexistant location" % history_dir,
-                             level=logging.ERROR)
-                    valid = False
-                else:
-                    try:
-                        if not os.path.samefile(data_folder, history_dir):
-                            self.log("DataFolder setting in %s (%s) and condor PER_JOB_HISTORY_DIR %s "
-                                     "do not match, these settings must match!" % (config_location,
-                                                                                   data_folder,
-                                                                                   history_dir),
-                                     level=logging.ERROR)
-                            valid = False
-                    except OSError as e:
-                        self.log("Error comparing DataFolder setting in %s (%s) and condor PER_JOB_HISTORY_DIR %s:\n%s"
-                                 % (config_location, data_folder, history_dir, e),
-                                 level=logging.ERROR)
-                        valid = False
-
-            # Per Gratia-126 DataFolder must end in / otherwise gratia won't find certinfo files
-            if not data_folder.endswith('/'):
-                self.log("DataFolder setting in %s must end in a /" % config_location,
-                         level=logging.ERROR)
-                valid = False
-
-        return valid
-
-    def _get_condor_history_dir(self, condor_config_val_bin):
-        cmd = [condor_config_val_bin, '-schedd', 'PER_JOB_HISTORY_DIR']
-        try:
-            process = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, encoding="latin-1")
-            (history_dir, errtext) = process.communicate()
-            if process.returncode != 0:
-                self.log("While checking gratia parameters: %s failed. Output follows:\n%s" % (condor_config_val_bin,
-                                                                                               errtext),
-                         level=logging.INFO)
-                return None
-        except OSError as err:
-            self.log("While checking gratia parameters: Error running %s: %s" % (condor_config_val_bin, str(err)),
-                     level=logging.INFO)
-            return None
-        history_dir = history_dir.strip()
-        if history_dir.startswith('Not defined'):
-            return None
-        return history_dir
```

Comment:
Given the discussion at the tech meeting yesterday, I think it'd be good to keep these for verifying the HTCondor-CE probe. I think the only changes needed would be to these functions:

- Using a `condor_ce_config_val_bin` (we can assume `/usr/bin/condor_ce_config_val`)
- Compare its value to the HTCondor-CE ProbeConfig